src/seedu/addressbook/AddressBook.java

```diff
@@ -275,7 +288,7 @@ private static void processProgramArgs(String[] args) {
         if(args.length == 0) {
             setupDefaultFileForStorage();
         }
-    }
+    }*/
```

Comment:
You don't need to comment this part, as you can revert back using git, if needed.